docker-compose.yml

```diff
@@ -39,7 +39,7 @@ services:
     command: ["start"]
     environment:
       - PORT=3000
-      - CONTENT_ASSET_PREFIX=http://localhost:3001
+      - CONTENT_ASSET_PREFIX=http://localhost:3001/about
```

Comment:
OOI is the PORT env var ignored here? 

Also i think with these path changes we don't need the apps running on different server ports to distinguish between apps

Reply:
Won't there be a conflict if the `fe-content-pages` service and the `fe-project` service both try to use 3000? Or is there some way to forward requests to 3000 based on the request URL, without adding a proxy service to the container?

Reply:
From memory, I set this up so that 3001 on the container maps to 3000 for the `fe-content-pages` service, but it was a long time ago so I'm not completely sure.

Reply:
Apologies I forget how this worked. Yes this setup is correct, the URL is injected to the HTML code which is loaded on the host and thus needs to have the PORT 3001 to distinguish between apps... my mistake. 

FWIW - we could add a nginx proxy with basic routing rules to run these services on 1 origin and split the traffic based on base_path now. but I don't think that really simplifies anything more than what we have.